administration/eslint.config.mjs

```diff
@@ -0,0 +1,219 @@
+import { fixupConfigRules, fixupPluginRules } from '@eslint/compat'
+import { FlatCompat } from '@eslint/eslintrc'
+import typescriptEslint from '@typescript-eslint/eslint-plugin'
+import tsParser from '@typescript-eslint/parser'
```

Comment:
:wrench: These should be replaced with the corresponding imports from 'typescript-eslint'. Same for all other imports/usages of '@typescript-eslint`.

Reply:
hmm 🤔 they are already from 'typescript-eslint' I didn't install them individually look at package.json